omop-fhir-translator/README.md

```diff
@@ -0,0 +1,72 @@
+# FHIR to OMOP translator
+
+## Startup
+
+Setup python env
+```
+python3 -m venv venv ; source venv/bin/activate
+pip install -r requirements.txt
+```
+### Setup Redivis API creds
+EDIT: This step isn't needed anymore. The actual data that is 
+used are the best files from 3 different datasets.
+
+go to https://redivis.com/
+make an account and generate an API token.
+
+Store the token in extract/.env file. Use extract/.example_env as a template.
+
+create the data folder and extract study data into it with:
+```
+mkdir data
+cd extract
+python extract/redivis_extract.py
+```
+downloads full tables  for concept, person, provider.
+
+## Setup Go dependancies and packages: 
+Install GO (at least Go 1.19): https://go.dev/dl/
+ 
+### Sifter / Lathe
+```
+go install github.com/bmeg/sifter@main
+go install github.com/bmeg/lathe@latest
+```
+### Iceberg Setup
+make sure that your iceberg is updated to FHIR version 5. If it is not 
+there will be validation errors. If it is updated, there might still be validation errors
+
+### Update path
+```
+export PATH=$PATH:$HOME/go/bin
+```
+### clean redivis data
+to get person.json files that are valid you need to run 
+data_cleaner.py and take the outputed file and replace it with data/person.json
+
+## Convert OMOP -> FHIR
+Build Snakefile
+```
+lathe plan plan_omop_to_fhir.yaml
```

Comment:
looks like one needs to `cd omop-fhir-translator/` first
```
$ lathe plan plan_omop_to_fhir.yaml
log: Error: failed to read config at path /Users/walsbr/bmeg/iceberg-schema-tools-mp/iceberg-schema-tools/plan_omop_to_fhir.yaml:
open /Users/walsbr/bmeg/iceberg-schema-tools-mp/iceberg-schema-tools/plan_omop_to_fhir.yaml: no such file or directory
```

Reply:
Yes, the root of the whole project lives inside the directory omop-fhir-translator so that It can be merged with the other tools

Reply:
```$ pwd
/Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/omop-fhir-translator
(venv) RNB10644:omop-fhir-translator walsbr$ lathe plan plan_omop_to_fhir.yaml
log: Sifter file count: 9
log: Lathe file count: 0
(venv) RNB10644:omop-fhir-translator walsbr$ snakemake -j -4
Building DAG of jobs...
MissingInputException in rule drug_exposure_omop_to_fhir in file /Users/walsbr/bmeg/iceberg-schema-tools-PR-8/iceberg-schema-tools/omop-fhir-translator/Snakefile, line 39:
Missing input files for rule drug_exposure_omop_to_fhir:
    output: output/drug_exposure_omop_to_fhir.MedicationRequest.MedicationRequest.json.gz, output/drug_exposure_omop_to_fhir.MedicationStatement.MedicationStatement.json.gz
    affected files:
        data_used/redivis_drug_exposure.json```